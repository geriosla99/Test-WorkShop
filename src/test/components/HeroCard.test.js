import React from "react";
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Enzyme from 'enzyme';
import { HeroCard } from "../../components/heroes/HeroCard";

describe('pruebas <HeroCard/>', () => {

    test('Validar la informacion de las tarjetas', () => {
        
        const id = 'dc-batman'
        const superhero = 'Batman'
        const alter_ego = 'Bruce Wayne'
        const first_appearance = 'Detective Comics #27'
        const characters = 'Bruce Wayne'

        Enzyme.configure({ adapter: new Adapter() })

        const wrapper = shallow( <HeroCard
            id = {id}
            superhero = {superhero}
            alter_ego = {alter_ego}
            first_appearance = {first_appearance}
            characters = {characters}
        />)

        const textParrafo = wrapper.find('h5').text().trim();
        console.log(textParrafo);

        expect(wrapper).toMatchSnapshot();
        expect(textParrafo).toBe('Batman')
    })
    
})
