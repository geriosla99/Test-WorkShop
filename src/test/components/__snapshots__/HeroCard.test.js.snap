// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas <HeroCard/> Validar la informacion de las tarjetas 1`] = `ShallowWrapper {}`;
